src/utils/validation.test.js

```diff
@@ -0,0 +1,19 @@
+import {
+  required,
+  requiredArray,
+} from './validation';
+
+describe('generateFileDefinitionBody', () => {
+  it('should validate correct simple values', () => {
+    expect(required(10)).toBe(undefined);
```

Comment:
> https://jestjs.io/docs/en/expect#tobeundefined
Yeah, but in this case, we do it in order to comply with how validation in react-final-form. In case the undefined is returned it means that the field is valid.

Reply:
> We can use .toBeUndefined(), it does the same thing, but it up to you to decide

Thanks for bringing this out. Let's use `.toBeUndefined()` from now on.